calico_rkt/calico_rkt.py

```diff
@@ -219,31 +237,154 @@ def _create_rules(id_):
     rules = Rules.from_json(rules_json)
     return rules
 
-def _apply_rules(profile_name, client):
+
+def _apply_default_rules(profile_name):
```

Comment:
I think apply_default_rules can be merged with create_default_rules, they're both pretty small and will probably always be called one after another. Rename the new one something like `assign_default_rules_to_profile`